dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
Can this be factored out into "the main-package proxy definition" and be where other future "things that talk to the main package" should go to live? Note that types here have to be carefully versioned compared to their framework-package brethren. LRP version 3 needs to be contactable by FXP versions 1-3.  Then we have a single place to put proxy/remote'd API definitions.

Reply:
I was wondering when we would get to the point of having the WinAppSDK version of "actxprxy" et. al. (Windows has a few dll's that aggregate proxy/stub implementations for efficiency sake).  Sounds like we are advocating to do this from the beginning, which seems good.

Reply:
@jonwis @EHO-makai I think I will need to ask you a few questions offline to get the full context on this. Also, while I understand it is a good idea, how am I supposed to complete this by Friday EOD? At the same time I'm also working on another critical piece of this feature (Protocol activation - PR is coming). But I definitely want to work on getting this done. I have filed a deliverable for it: #1349